tests/ctl/conftest.py

```diff
@@ -65,3 +67,12 @@ def db():
 
     yield session
     session.close()
+
+
+@pytest.fixture(scope="function", autouse=True)
+def load_default_data_uses(db):
+    for data_use in DEFAULT_TAXONOMY.data_use:
+        # weirdly, only in some test scenarios, we already have the default taxonomy
+        # loaded, in which case the create will throw an error. so we first check existence.
```

Comment:
Is it this again? https://github.com/ethyca/fides/issues/2016

Reply:
yup, i believe so. i could remove this comment if you think it's just confusing. i'd just copied it over from the original location of the fixture, when i first wrote it.

Reply:
Better yet would also be to not-remove the default taxonomy data too — we could definitely do this with some less blunt removal code

Reply:
ah ok we could clarify here since we know the cause until we make time to fix